### ISSUE TITLE
Issues getting this to run

### ISSUE THREAD

**Issue by patabongo** [gryphonshafer/Delphi-Scrape#1]
Hi, thanks for writing this! I've been trying to use it for an archive project and running into some issues.

The first was that I initially got the error

`catch block requires a (VAR) at /app/Delphi.pm line 289, near "catch {"`

I was able to deal with that by changing the line to:

`    catch ($e) {}`

Unfortunately I'm now getting a different error:

`/usr/local/bin/phantomjs: 2: Syntax error: word unexpected (expecting ")")`
`Selenium server did not return proper status at /root/.perlbrew/libs/perl-5.40.0@local/lib/perl5/Selenium/Remote/Driver.pm line 547.`

and I'm not sure how to debug that. From looking around, it seems "word unexpected" errors can come from OS or architecture incompatibilities - I've been trying to run Docker on Raspberry Pi OS and ARM64. Do you mind saying which architecture and OS you were using?

**Comment by gryphonshafer** [comment]
I ran it in a Debian container (described in the [Dockerfile](../blob/master/Dockerfile)) on a Debian host.

The code here is not exactly abandonware, but it's pretty close. I built it to help out a group wanting to migrate off Delphi several years ago. Unfortunately, I don't have time to maintain it. It heavily relies on dependencies like PhantomJS, and fortunately there's not a lot of code in the project itself, so an approach to discovering the current blocking bug may be to try to determine the line in the project code where the error comes from.
